providers/ipstack.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ipstack",
+  "services": [
+    {
+      "id": "default",
+      "baseUrl": "{HTTP_SCHEME}://api.ipstack.com/"
+    }
+  ],
+  "defaultService": "default",
+  "securitySchemes": [
+    {
+      "id": "apikey",
+      "type": "apiKey",
+      "in": "query",
+      "name": "access_key"
+    }
+  ],
+  "parameters": [
+    {
+      "name": "HTTP_SCHEME",
+      "description": "HTTPS or HTTP scheme. The scheme identifies the protocol to be used to access the IP Stack API. Only HTTP scheme is allowed in case of IP Stack FREE subscription."
```

Comment:
Wow, that's some weird monetization scheme. Would be useful to have explicit example (`"http" or "https"`) and mention what the default is.

Can we actually set the default?

Reply:
I added explicit example and set the default to "https".